_posts/2017-11-03-2017-crystal-survey-2017-results.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Crystal Survey 2017 Results"
+author: sdogruyol
+summary: ""
+thumbnail: ""
+---
+
+This year, the Crystal team launched the first official [State of Crystal survey](https://crystal-lang.org/2017/07/31/launching-the-2017-state-of-crystal-survey.html). We're amazed by the fantastic feedback from the community.
+
+We've received ***757*** responses, thanks a lot to everyone who took the time to respond.  
+
+We've tried our best to read through all of your comments and suggestions. Without further ado, let's take a look at the results.
+
+
+![%19.9 Google or other search engines, %19.4 HackerNews, %15.6 Reddit %10.9 Twitter](https://cdn.pbrd.co/images/GP30DmW.png
+  "%19.9 Google or other search engines, %19.4 HackerNews, %15.6 Reddit %10.9 Twitter")
+(hover for more info)
+
+We wanted to learn how did you discover Crystal, it's interesting to see that a lot our respondents find Crystal from search engines. HackerNews, Reddit and Twitter shows the power of social media.
+
+# Using Crystal
+
+![%47,2 Yes, I' using Crystal for hobby projects, %32.1 No, but I' planning to use Crystal, %10.7 No, I'm not using Crystal, %9.9 Yes, I'm Using Crystal in production](https://cdn.pbrd.co/images/GP33qmx.png
+"%47,2 Yes, I' using Crystal for hobby projects, %32.1 No, but I' planning to use Crystal, %10.7 No, I'm not using Crystal, %9.9 Yes, I'm Using Crystal in production")
+(hover for more info)
+
+It's amazing to see a lot people already using Crystal. Nearly half of our respondents use Crystal in for hobby projects. Another interesting point is that nearly 1 out of 10 are running in production.
+
+![%34.7 Less than a month, %19 More than 12 months, % 18.9 1-3 Months, %15.2 3-6 Months](https://cdn.pbrd.co/images/GP35Uvi.png
+  "%34.7 Less than a month, %19 More than 12 months, % 18.9 1-3 Months, %15.2 3-6 Months")
+(hover for more info)
+
+Crystal community is growing rapidly. We've a steady growing user base with a lot of new Crystal developers.
+
+![%63.1 Less than 1.000, %23 1000 to 10000 lines, %11.5 Not sure, %2.+ 10000 to 100000 lines](https://cdn.pbrd.co/images/GP38e1h.png
+"%63.1 Less than 1.000, %23 1000 to 10000 lines, %11.5 Not sure, %2.+ 10000 to 100000 line")
+(hover for more info)
+
+Most of the current Crystal projects are less than 1000 lines, However there are also 17 projects which are in between 10000 and 100000 lines.
+
+![%46.6 Yes, %32.9 Not sure, %11.9 No, %8.9 Not applicable](https://cdn.pbrd.co/images/GP39tRi.png
+  "%46.6 Yes, %32.9 Not sure, %11.9 No, %8.9 Not applicable")
+(hover for more info)
+
+Crystal is a beauty, we know that a lot of you are excited to use it for your daily work :)

```

Comment:
We can combine these results with the previous couple of questions and hypothesize that for many, Crystal is good enough to knock out small (and performant) scripts that help at work without entailing the risk of a fully fledged, mission critical production system.